skrub/_dataframe/tests/test_polars.py

```diff
@@ -29,7 +31,11 @@
 
 def test_join():
     joined = join(left=main, right=main, left_on="movieId", right_on="movieId")
-    expected = main.join(main, on="movieId", how="left", coalesce=True)
+    if "coalesce" in inspect.signature(main.join).parameters:
```

Comment:
@jeromedockes I think that we need to make an extra change here as well.

Reply:
which one?

Reply:
I mean we did not to this change in `main` when merging the previous PR.

Reply:
ah right -- we did it in the skrub code but not in the test

Reply:
Since we are going to merge this PR, let's doing it here. This is a minimal change.